ext/calendar/easter.c

```diff
@@ -48,6 +48,11 @@ static void _cal_easter(INTERNAL_FUNCTION_PARAMETERS, bool gm)
 		}
 	}
 
+	if (year < 0 || year > (ZEND_LONG_MAX - 1)) {
```

Comment:
The computation `dom = (year + (year/4) - (year/100) + (year/400)) % 7` can still overflow.
Also, year 0 does not exist.